modules/virt-example-snippet-nmstate-multiple-interfaces.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-snippet-nmstate-multiple-interfaces_{context}"]
+= Example snippet: Multiple interfaces in the same Policy
+
+You can create multiple interfaces in the same Policy. These interfaces can reference each other, allowing you to build and deploy a network configuration using a single Policy manifest.
+
+The following example snippet creates a bond (`bond10`) across two NICs and a Linux bridge (`br1`) that connects to the bond.
```

Comment:
I don't think that "snippet" is necessary when you're already saying "example".

Reply:
Despite what these changes may suggest, I actually don't much like the term 'snippet'. However I do think it's important to be very specific here between what is a complete example configuration (also featured in the assembly), and what is only a part of a configuration. nmstate is pretty powerful and these policies can alter the network of the whole cluster so I guess this is my prudence showing :)

Reply:
Cleaned up some of the flagrant usage. Now only used in leading para specific to the config